population/create_glpi_computer.py

```diff
@@ -106,6 +112,8 @@ def main() -> None:
     PUT = args.put
     global COMPUTER_ID
     COMPUTER_ID = args.computer_id
+    global OVERWRITE
```

Comment:
Nitpick, but this probably doesn't need to be global, it can just be passed into post_to_glpi, and avoid a global where not needed

Reply:
Good point, changed in all 3 files.